src/fastcs_odin/frame_processor.py

```diff
@@ -95,6 +98,27 @@ def __dataset_parameter(param: OdinParameter):
 
         return await super().initialise()
 
+    async def _create_commands(self):
+        command_path = f"command/{self.path[-1].lower()}"
+        command_response = await self.connection.get(
+            f"{self._api_prefix}/{command_path}/allowed"
+        )
+
+        try:
+            commands = AllowedCommandResponse.model_validate(command_response)
+            for command in commands.allowed:
+                self.construct_command(command, command_path)
+        except ValidationError:
+            pass
+
+    def construct_command(self, command_name, command_path):
```

Comment:
This should also be "private"
```suggestion
    def _construct_command(self, command_name, command_path):
```